examples/llava/clip.h

```diff
@@ -55,6 +59,7 @@ CLIP_API const int32_t * clip_image_grid(const struct clip_ctx * ctx);
 CLIP_API int clip_n_patches    (const struct clip_ctx * ctx);
 CLIP_API int clip_n_mmproj_embd(const struct clip_ctx * ctx);
 
+CLIP_API struct load_image_size * load_image_size_init();
```

Comment:
Inconsistent name, should be `clip_image_size_init`

Reply:
ok